### PR TITLE
Dashboard: Discard the first few data points from metrics

### LINKED ISSUE #379: dashboard occasionally shows a huge first datapoint
While traffic is going through linkerd, the first time you load the dashboard, the first data point on the request volume graph is sometimes really high. I think this a function of how the aggregation code works for metrics.json. One short-term workaround might be to ignore the first value.

### PR DESCRIPTION
**Pull request by rmars**
... as the request count we get back from the metrics endpoint seems to be very high for the first few data points, causing a spike in the graph

Fixes #379 (hackily)

### CODE REVIEW

**Comment by esbie** [comment]
⭐  looks good code-wise. I'd be interested to understand why this happens

**Comment by olix0r** [comment]
My understanding is that we should only have to drop _the first_ data point (since it is absolute and not a delta to the prior). Is this not the case?

**Comment by rmars** [comment]
When I tested it out, it looked like more than the first data point had an unusually large value - so I left out the first couple to be sure. Let me check again. It does mean a more seconds of waiting for the graph after loading the dashboard page.

**Comment by rmars** [comment]
Yep, checked again... the first 4 ish values are way too high.

**Comment by olix0r** [comment]
:star: very odd... i'd like to understand why that happens but this seems like a good workaround

File: admin/src/main/resources/io/buoyant/admin/js/combined_client_graph.js [hunk 1]
```diff
@@ -45,10 +45,19 @@ var CombinedClientGraph = (function() {
...
     chart.setMetrics(desiredMetrics, timeseriesParams, true);
 
-    metricsCollector.registerListener(function(data) {
-      var filteredData = Query.filter(query, data.specific);
-      chart.updateMetrics(filteredData);
-    }, function(metrics) { return Query.filter(query, metrics); });
+    var count = 0;
+    var metricsListener = function(data) {
+      if (count < 5) {
+        // Hacky bug fix: discard the first few data points to fix the issue
```
Comment (adleong) [inline admin/src/main/resources/io/buoyant/admin/js/combined_client_graph.js]: Might be worth opening an issue for investigating why this happens.  Even if we never do it, at least there's a record documenting why we drop data points.

**Comment by adleong** [comment]
:star: ▶️

File: admin/src/main/resources/io/buoyant/admin/js/combined_client_graph.js (hunk 1 above)
Comment (rmars) [inline admin/src/main/resources/io/buoyant/admin/js/combined_client_graph.js]: Good point! Created #485, I'll add it here.

**Pull request merged**